src/common/ceph_context.h

```diff
@@ -151,6 +151,9 @@ class CephContext {
   ceph::logging::Log *_log;
 #ifdef HAVE_BREAKPAD
   std::unique_ptr<google_breakpad::ExceptionHandler> _ex_handler;
+  static_assert(sizeof(std::unique_ptr<google_breakpad::ExceptionHandler>) == sizeof(std::unique_ptr<char>));
+#else
+  std::unique_ptr<char> _ex_handler;
```

Comment:
I would comment it's has its purpose. Perhaps `[[maybe_unused]]`` would be enough to communicate "do not remove!" to humans.